conf/messages

```diff
@@ -115,6 +115,9 @@ error.plugin.notSponge          =   Project is marked as a Sponge Plugin and new
 error.plugin.notForge           =   Project is marked as a Forge Mod and new version contains no dependency to Forge.
 error.plugin.fileName           =   Filename already in use. Please rename your file and try again.
 
+error.spongeauth.auth           =   Couldn't connect to SpongeAuth.
+error.spongeauth.parse          =   That name is invalid, please choose a CamelCase name like <i>ExampleOrg</i>.
```

Comment:
Why CamelCase?

Reply:
Yeah, just shouldn't have spaces.

Reply:
`That name is invalid, please choose a name without spaces or symbols`?